v2/pkg/subscraping/sources/dnsdb/dnsdb.go

```diff
@@ -58,42 +72,86 @@ func (s *Source) Run(ctx context.Context, domain string, session *subscraping.Se
 		}
 
 		sourceName := s.Name()
+
```

Comment:
Minor, but you may as well do the path construction code later because it won't be used immediately and the function can return with a failure before it is used.

Reply:
Ok moved.